components/aws/sagemaker/test/integration/resources/simple_mnist_training/simple_mnist_training.py

```diff
@@ -0,0 +1,79 @@
+import kfp
```

Comment:
Are we taking this from the samples directory?

Reply:
yes, going to change this in next iteration to make the pipeline definition more generic, this is intended as sample